recipes/torchvision-extra-decoders/recipe.yaml

```diff
@@ -0,0 +1,79 @@
+context:
+  name: torchvision-extra-decoders
+  version: "0.0.2"
```

Comment:
Where did you get the number from? I had the impression it was calculated from the number of commits

Reply:
I added comments for this.